### PR TITLE
Style cart

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- add styling for black button on product and cart pages
- fix full-height class for pages with full-width page title.
- style cart

### CODE REVIEW

File: templates/cart.liquid [hunk 1]
```diff
@@ -1,57 +1,64 @@
...
         </div>
-      </div>
 
-      <div class="grid-item item-s-12 item-l-6 offset-l-6 no-gutter grid-row">
-        <input class="button item-s-6 font-uppercase" type="submit" id="update-cart" name="update" value="Update Cart" title="Update Cart" />
-        <input class="button item-s-6 font-uppercase" type="submit" id="cart-checkout" name="checkout" value="Checkout" />
+        <div class="grid-item no-gutter grid-row border-top">
+          <input class="button item-s-6 font-uppercase font-mono font-size-mid padding-top-basic padding-bottom-basic" type="submit" id="update-cart" name="update" value="Update Cart" title="Update Cart" />
+          <input class="button button-black item-s-6 font-uppercase font-mono font-size-mid padding-top-basic padding-bottom-basic" type="submit" id="cart-checkout" name="checkout" value="Checkout" />
+        </div>
```
Comment (varffs) [inline templates/cart.liquid:61]: Empty cart state needs styling tho

File: templates/cart.liquid [hunk 2]
```diff
@@ -1,57 +1,64 @@
...
-            <div class="margin-bottom-micro">
-              <a href="{{ item.product.url }}">{{ item.product.title }}</a>
+    <form action="/cart" method="post" id="cart" class="cart full-height-after-title">
+      <div id="cart-items-column" class="grid-item item-s-12 item-l-6 flex-grow no-gutter grid-column border-left-half justify-between">
+        <div class="grid-item no-gutter flex-row font-size-small">
+          {% for item in cart.items %}
+          {% assign variant_price = item.price | money %}
+          <div id="cart-item-{{item.variant.id}}" class="cart-item grid-item item-s-12 padding-top-small padding-bottom-small no-gutter grid-row align-items-center border-bottom">
+            <div class="grid-item item-s-2 item-l-2 text-align-center">
+              <input type="text" size="4" pattern="[0-9]*" name="updates[{{item.variant.id}}]" id="quantity-{{ item.variant.id }}" class="form-element quantity" value="{{ item.quantity }}" min="1" onfocus="this.select();"/>
```
Comment (varffs) [inline templates/cart.liquid]: should this limit to 2 for likelyhood and also style if accidentally filled?

**Review by varffs — CHANGES_REQUESTED** [review]
Just something simple for the empty state. I would do it but feel easier for you to do quickly with the right look and feel

File: templates/cart.liquid (hunk 1 above)
Comment (mikeyrayvon) [reply templates/cart.liquid:61]: oh you're so right

File: templates/cart.liquid (hunk 2 above)
Comment (mikeyrayvon) [reply templates/cart.liquid]: yea limit to 2 is good.

style if accidentally filled? could you clarify

Comment (varffs) [reply templates/cart.liquid]: just if the user held the button down or something and entered 4 digits it looked bad.

**Pull request merged**